pypeit/wavetilts.py

```diff
@@ -29,9 +29,13 @@ class WaveTilts(datamodel.DataContainer):
       although they can be None (but shouldn't be)
 
     """
+    # TODO: Is minimum version ever used? Can we get rid of it? (I
```

Comment:
We decided to only *try* it for a few containers.  This is not one of them.
So, please remove.  Thanks.

Reply:
Done.